scripts/mongo/common.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+foreach (array(__DIR__ . '/../../../../autoload.php', __DIR__ . '/../../vendor/autoload.php') as $file) {
+    if (file_exists($file)) {
+        define('TRIPOD_COMPOSER_INSTALL', $file);
+        break;
+    }
+}
+
+if (!defined('TRIPOD_COMPOSER_INSTALL')) {
+    die(
+        'You need to set up the project dependencies using the following commands:' . PHP_EOL .
+        'wget http://getcomposer.org/composer.phar' . PHP_EOL .
```

Comment:
It'll do the same job so not really that fussed, but should we use the getcomposer.org way of doing it?
`curl -sS https://getcomposer.org/installer | php`

Or maybe just run `ant init` to do it for us?